redis/src/sentinel.rs

```diff
@@ -301,7 +301,7 @@ fn find_valid_master(
 #[cfg(feature = "aio")]
 async fn async_check_role(connection_info: &ConnectionInfo, target_role: &str) -> bool {
     if let Ok(client) = Client::open(connection_info.clone()) {
-        if let Ok(mut conn) = client.get_multiplexed_async_connection(None).await {
+        if let Ok(mut conn) = client.get_multiplexed_async_connection(None, None).await {
```

Comment:
I would have refactor this and add `client.get_multiplexed_async_connection_with_notifier(...)` rather than adding a second param to the existing function

Reply:
Tried that, I created much more mess